### DISCUSSION TITLE
How the first DAG run is triggered by CronTriggerTimetable

### DISCUSSION THREAD

**Discussion by kimminw00** [apache/airflow#35647] [Q&A]
```Python3
import pendulum

from airflow import DAG
from airflow.operators.bash import BashOperator

from airflow.timetables.trigger import CronTriggerTimetable

with DAG(
    dag_id="example_test_schedule",
    schedule=CronTriggerTimetable("0 12 * * *", timezone="Asia/Seoul"),
    start_date=pendulum.datetime(2021, 1, 1, tz="Asia/Seoul"),
    catchup=False,
    tags=["example"],
) as dag:
    bash_task = BashOperator(
        task_id="bash_task",
        bash_command='echo "hello world"',
    )
```
If I enable the DAG using the CronTriggerTimetable at 3PM on November 15th, CronTriggerTimetable immediately trigger a new DAG run which is supposed to trigger at 12AM on November 15th.

> Here is an example showing how the first DAG run is triggered. Supposes there is a cron expression @daily or 0 0 * * *, which is aimed to run at 12AM every day. If you enable DAGs using the two timetables at 3PM on January 31st, [CronTriggerTimetable](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#crontriggertimetable) will trigger a new DAG run at 12AM on February 1st. [CronDataIntervalTimetable](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#crondataintervaltimetable), on the other hand, will immediately trigger a new DAG run which is supposed to trigger at 12AM on January 31st if the DAG had been enabled beforehand. 

From the above lines, I think that CronTriggerTimetable should trigger a new DAG run at 12AM on November 16th and not immediately trigger a new DAG run which is supposed to trigger at 12AM on November 15th. Is there something I missed?

### Environment
Airflow 2.7.2

### Reference
https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#the-time-when-a-dag-run-is-triggered

**Comment by jojaeng2** [comment]
hello, @kimminw00 
Did you solve this problem by any chance? I wanted the same usability, so I used `CronTriggerTimetable`, but if I change it to the enable state, it will be executed once. Please help me if you solved it.

**Reply by kimminw00** [reply]
I haven't solved it yet! But I keep looking for solutions.

**Reply by jojaeng2** [reply]
Hello, I'm going to solve this problem and share it with you, 
1. Set start_date to now. 
2. Use cron expressions using CronTriggerTimable

I think it will be helpful.

**Reply by jojaeng2** [reply]
In airflow 2.8.0, I solved it this way. I hope it helps

**Reply by engg14000** [reply]
Thanks @jojaeng2! The workaround you suggested helps! However, the expected behavior is that CronTriggerTimetable should skip the immediate run, whereas CronDataIntervalTimetable will perform an immediate run.

**Reply by jojaeng2** [reply]
Hello @engg14000 , I don't understand the comments you left, can you explain a little more?
